dspace-api/pom.xml

```diff
@@ -609,6 +609,12 @@
         <dependency>
             <groupId>org.apache.tika</groupId>
             <artifactId>tika-parsers-standard-package</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>xml-apis</groupId>
+                    <artifactId>xml-apis</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Why do we need to exclude this? How is this related to the issue this PR is solving?